### DISCUSSION TITLE
AMS METRICS

### DISCUSSION THREAD

**Discussion by IVEN2024** [ant-media/Ant-Media-Server#6729] [General]
How to get AMS metrics
 stream quality , stream mbps , stream codec etc.   ??

**Comment by rahul78275** [comment]
Hello @IVEN2024 

Have a look at these lines [565](https://github.com/ant-media/StreamApp/blob/a8821c1538c16dc0f8a585c487402b1ac798bcb7/src/main/webapp/samples/publish_webrtc.html#L565) and [599](https://github.com/ant-media/StreamApp/blob/a8821c1538c16dc0f8a585c487402b1ac798bcb7/src/main/webapp/samples/publish_webrtc.html#L599)

Is this helpful ?

**Comment by muratugureminoglu** [comment]
Hi @IVEN2024 

You can collect some metrics by using Kafka and analytics logs. 

For Kafka: https://antmedia.io/docs/guides/monitoring/monitoring-ams-with-grafana/#which-data-can-you-access-from-ant-media

For Analytics: https://antmedia.io/docs/guides/monitoring/monitor-ant-media-server-statistics-with-new-relic/

Please let me know If you have further questions. 

Regards.
